client/src/annoMatrix/annoMatrix.ts

```diff
@@ -203,23 +218,23 @@ export default class AnnoMatrix {
   /**
    ** General utility methods
    **/
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-  base() {
+  base(): AnnoMatrix {
     /*
     return the base of view, or `this` if not a view.
     */
-    // eslint-disable-next-line @typescript-eslint/no-this-alias --- FIXME: disabled temporarily on migrate to TS.
+    // eslint-disable-next-line @typescript-eslint/no-this-alias --- TODO revist: Possibly remove need for aliasing here, see related comments below on L226.
     let annoMatrix = this;
+    // @ts-expect-error ts-migrate --- TODO revisit:
+    // Compiler is complaining that this could be instantiated with an arbritrary type which AnnoMatrix | null cannot be
+    // assigned to. Possibly move method to recursion pattern to resolve this and issue above (L224).
     while (annoMatrix.isView) annoMatrix = annoMatrix.viewOf;
     return annoMatrix;
   }
 
   /**
    ** Load / read interfaces
    **/
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'field' implicitly has an 'any' type.
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-  fetch(field, q) {
+  fetch(field: Field, q: Query): Promise<Dataframe | undefined> {
```

Comment:
fetch() returns an instance of `class DataFrame`, not the referenced schema sub-object.  Should return `Promise<any>` until types for `util/dataframe` are available.

Reply:
See comment in query.ts about fixing the definition of Query.

I recommend Query be a singleton query, and this function be redefined as:
> fetch(field: Field, q: Query | Query[])

prefetch() and _fetch() need the same edit.